rascal-lsp/src/main/rascal/util/LanguageServer.rsc

```diff
@@ -130,13 +197,28 @@ data LanguageService
     | implementer(Implementer implementer)
     ;
 
+@deprecated{Please use ((builder)) or ((analyzer))}
+@synopsis{A summarizer collects information for later use in interactive IDE features.}
+LanguageService summarizer(Summarizer summarizer) {
```

Comment:
we also need to chain through the KW params. (`bool providesDocumentation = true` etc)

Reply:
Fixed